### DISCUSSION TITLE
Running HuggingFace XTTS space

### DISCUSSION THREAD

**Discussion by nitinmukesh** [idiap/coqui-ai-TTS#158] [Q&A] [answered]
Hello,

I need help to understand how to install on Windows 11 with all languages and WebUI like
https://huggingface.co/spaces/coqui/xtts

I have seen this section
https://github.com/idiap/coqui-ai-TTS#optional-dependencies

is this correct to install with all languages
pip install coqui-tts[server,all]

Also after installation, how to launch webui

**Comment by eginhard** [comment, marked as answer]
You need to download the code for that space from here, it's not part of this repo: https://huggingface.co/spaces/coqui/xtts/tree/main
